test/config-next/ra.json

```diff
@@ -3,8 +3,8 @@
     "rateLimitPoliciesFilename": "test/rate-limit-policies.yml",
     "maxConcurrentRPCServerRequests": 16,
     "maxContactsPerRegistration": 100,
-    "dnsTries": 2,
-    "debugAddr": "localhost:8002",
+    "dnsTries": 3,
+    "debugAddr": "0.0.0.0:8002",
```

Comment:
You changed the other `debugAddr`'s to use `":8002"`, is it worth dropping the "0.0.0.0" from this one to keep them consistent?